ext/zlib/tests/zlib_get_coding_type.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+string zlib_get_coding_type ( void );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (phpversion() < "5.6.0") {
+    die('SKIP php version so lower.');
+}
+?>
+--FILE--
+<?php
+$compressionCoding = zlib_get_coding_type();
+$codingType = "get zlib type error";
+    
+if((!$compressionCoding) || ("gzip" == $compressionCoding) || ("deflate" == $compressionCoding)){
+  $codingType = "okey";
+}
+
+print($codingType);
+?>
+--CLEAN--
+<?php
```

Comment:
The --CLEAN-- section is not needed for variable unsets, its a separate generated file after the phpt is executed. It is commonly used to remove temporary files and similar